documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/blang/semver"
```

Comment:
I'd try to keep this simple, as it's primarily an example and depends on experimental features.

I'd just make the change and only support the new way.